src/utils/LiftUtils.ts

```diff
@@ -22,33 +22,33 @@ export const toValidSet = (repsWithRpe: string): Set => {
   return { reps: validRepNumber, rpe };
 };
 
-export const allRepsAreEqualAndWithoutRpes = (sets: Set[]) =>
+export const allRepsAreEqualAndWithoutRpes = (sets: ReadonlyArray<Set>) =>
```

Comment:
just realized that it is quite confusing to have a custom type called `Set`, since there is a JS type `Set`, but I guess we can't  do anything about this :)